Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -1523,8 +1540,7 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Searches for alerts that happened in the past day with Malware
-        category where the host IP is the client IP of the current alert.
+      description: Searches for alerts that happened in the past day with Malware category where the host IP is the client IP of the current alert.
```

Comment:
Searches for Malware category alerts that occurred in the past day with host IP equal to the client IP of the current alert.